modules/ingestor/src/graph/organization.rs

```diff
@@ -78,7 +78,10 @@ impl Graph {
         let name = name.into();
         let information = information.into();
 
-        if let Some(found) = self.get_organization_by_name(&name, &tx).await? {
+        if let Some(found) = self
+            .get_organization_by_name(&name, Transactional::None)
```

Comment:
That isn't about readability, but actually changing logic, isn't it. There's a difference between re-using the existing transaction (or none) vs explicitly not using a transaction.

Reply:
> vs explicitly not using a transaction

yeah so when this is explicit then is more easy to read and understand 

> In computer programming, [readability](https://en.wikipedia.org/wiki/Readability) refers to the ease with which a human reader can comprehend the purpose,

[link](https://en.wikipedia.org/wiki/Computer_programming#:~:text=the%20aforementioned%20attributes.-,Readability%20of%20source%20code,usability%20and%20most%20importantly%20maintainability.)

Reply:
Yea, but it's changing the behavior of the application. And I don't think that's the correction action here.

Reply:
I'm ok closing the PR, also this is not solving the problem with the upload :+1: